tools/internal_ci/linux/grpc_e2e_performance_gke.sh

```diff
@@ -130,16 +130,6 @@ configLangArgs8core+=( -l c++ )
 configLangArgs32core+=( -l c++ )
 runnerLangArgs+=( -l "cxx:${GRPC_CORE_REPO}:${GRPC_CORE_COMMIT}" )
 
-# dotnet
```

Comment:
Please comment out instead of removing. That way we remember what has been disabled and could be reenabled or removed.

Reply:
Done